synapse/tests/test_cortex.py

```diff
@@ -901,6 +901,46 @@ def runtufobydefault(self, core):
 
 class CortexTest(SynTest):
 
+    def test_cortex_datamodel_runt_consistency(self):
+        with self.getRamCore() as core:
+
+            nodes = core.getTufosByProp('syn:form')
+            for node in nodes:
+                self.isin('syn:form:ptype', node[1])
+                if 'syn:form:doc' in node[1]:
+                    self.isinstance(node[1].get('syn:form:doc'), s_compat.strtypes)
+                if 'syn:form:local' in node[1]:
+                    self.isinstance(node[1].get('syn:form:local'), int)
+
+            nodes = core.getTufosByProp('syn:prop')
+            for node in nodes:
+                self.isin('syn:prop:form', node[1])
+                if 'syn:prop:glob' in node[1]:
+                    continue
+                self.isin('syn:prop:req', node[1])
+                self.isin('syn:prop:ptype', node[1])
+                if 'syn:prop:doc' in node[1]:
+                    self.isinstance(node[1].get('syn:prop:doc'), s_compat.strtypes)
```

Comment:
These make me think maybe (eventually) having self.isint() self.isstr() type helpers might be nice too.  Thoughts?

Reply:
```isint()``` is pretty close to ```isin()```.  This was used mainly because of 2.7 and unicode strings vs. str strings.  It would be trivial to add helpers like that.